sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLQuerySuite.scala

```diff
@@ -1399,52 +1399,6 @@ class SQLQuerySuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
     }
   }
 
-  test(
```

Comment:
If we plan to bring this syntax back after fixing the problem, should we let it ignore and add TODO instead of completely removing these tests?